cg/cli/workflow/taxprofiler/base.py

```diff
@@ -125,3 +137,5 @@ def run(
     except Exception as error:
         LOG.error(f"Could not run analysis: {error}")
         raise click.Abort() from error
+    if not dry_run:
```

Comment:
Currently we only interact with trailblazer for tower analyses (not when using nextflow). We should make sure that every we interact with trailblazer we only do it when using tower something like:
```suggestion
    if not dry_run and with_tower:
```
Similar when accessing the tower_id for re- runs. I believe this is currently not handled in rnafusion. 
Could you test your branch for running taxprofiler with nextflow from-start and with resume, just to make sure we are handling those?